### PR TITLE
Rename SurveyContact to Dialogue

### PR DESCRIPTION
**Pull request by catkins**
Renamed `SurveyContact` to `Dialogue` to better fit the domain. Need to clear the local by deleting the app to get it working.

### CODE REVIEW

File: app/src/main/java/com/android/smap/api/models/Contact.java [hunk 1]
```diff
@@ -57,8 +57,8 @@ public static Contact findById(Long id) {
 		return new Select().from(Contact.class).execute(); 
 	}
 	
-	public List<SurveyContact> getSurveyContacts() {
-		return getMany(SurveyContact.class, "contact_id");
+	public List<Dialogue> getSurveyContacts() {
```
Comment (swellsdev) [inline app/src/main/java/com/android/smap/api/models/Contact.java:60]: getDialogues()?

Comment (catkins) [inline app/src/main/java/com/android/smap/api/models/Contact.java:60]: Yep

File: app/src/main/java/com/android/smap/api/models/Distribution.java [hunk 2]
```diff
@@ -45,12 +45,12 @@ public void setSurvey(Survey survey) {
         this.survey = survey;
     }
 
-    public List<SurveyContact> getSurveyContacts() {
-        List<SurveyContact> contacts = getMany(SurveyContact.class, "distribution_id");
-        Collections.sort(contacts,new Comparator<SurveyContact>() {
+    public List<Dialogue> getSurveyContacts() {
```
Comment (swellsdev) [inline app/src/main/java/com/android/smap/api/models/Distribution.java:48]: getDialogues()?

**Comment by catkins** [comment]
Didn't give me a chance to tidy up those issues :p are you gonna be digging around there tonight? If so can you fix them up?

**Pull request merged**